driver/src/main/java/com/code_intelligence/jazzer/junit/BUILD.bazel

```diff
@@ -1,24 +1,42 @@
 load("@com_github_johnynek_bazel_jar_jar//:jar_jar.bzl", "jar_jar")
 load("@fmeum_rules_jni//jni:defs.bzl", "java_jni_library")
+load("@rules_jvm_external//:defs.bzl", "javadoc")
 load("//bazel:compat.bzl", "SKIP_ON_WINDOWS")
 load("//bazel:jar.bzl", "strip_jar")
 load("//:maven.bzl", "JAZZER_JUNIT_MAVEN_DEPS")
 load("//sanitizers:sanitizers.bzl", "SANITIZER_CLASSES")
 
-# The Jazzer JUnit integration without its dependencies declared in the pom.xml.
-jar_jar(
+# The Jazzer JUnit integration without its dependencies declared in the pom.xml, together with the unshaded sources.
+java_import(
     name = "junit",
+    jars = ["junit_shaded.jar"],
+    srcjar = "junit_unshaded_deploy-src.jar",
+    visibility = ["//visibility:public"],
+)
+
+javadoc(
+    name = "junit-docs",
```

Comment:
The `@FuzzTest` Javadoc seems outdated.

Reply:
Yes, I want to make one more breaking change to the API before I update it.